<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog7</class>
 <widget class="QDialog" name="Dialog7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BuyButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>281</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Size1">
        <property name="text">
         <string>35</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Size2">
        <property name="text">
         <string>36</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Size3">
        <property name="text">
         <string>37</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Size4">
        <property name="text">
         <string>38</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Size5">
        <property name="text">
         <string>39</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="Size6">
        <property name="text">
         <string>40</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Size7">
        <property name="text">
         <string>41</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Size8">
        <property name="text">
         <string>42</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Size9">
        <property name="text">
         <string>43</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
